### ISSUE TITLE
Unable to Activate Brickroad Elements

### ISSUE THREAD

**Issue by cfrishe** [Desertsnowman/brickroad#1]
When attempting to activate the Brickroad Elements Plugin I get the following Error:

Parse error: syntax error, unexpected '<' in /plugins/brickroad-elements/includes/functions-find-partner.php on line 2

**Comment by DavidCramer** [comment]
In the functions/PHP tab, don't start with <?php

**Comment by cfrishe** [comment]
The error appeared after I loaded existing Caldera elements. Is this error specific to an element that I have created?

**Comment by DavidCramer** [comment]
Ok, do you have Caldera Engine running concurrently?

**Comment by cfrishe** [comment]
No, I exported my elements, disabled Caldera Engine, enabled Brickroad and then imported the elements. The elements imported but I thought the issue I was seeing was related to the Brickroad rather than my existing code. It looks like issues with inactive elements are preventing any other elements from activating. I am looking through the bad inactive bad code now to see if I can resolve the issues there.

**Comment by DavidCramer** [comment]
Could you send me the export file to david@digilab.co.za ?

**Comment by cfrishe** [comment]
Sending now.

**Comment by DavidCramer** [comment]
Thanks. As I mentioned, the code base is 4 years old. So expect some issues.

**Comment by cfrishe** [comment]
No worries, I am just thrilled that we are able to work through them.

**Comment by cfrishe** [comment]
It appears that there is an issue with using "-" in a slug name for a widget element. If I replace the dash with an underscore, it seems to resolve the issue.

**Comment by DavidCramer** [comment]
Nice. I'll take a better look in the morning. I'm only on Mobile, so it's tricky to see what's happening.

**Comment by cfrishe** [comment]
That seems to fix it. Removing the one element that had brackets on the PHP and changing the slug for widget elements to not use dashes appears to resolve things so everything can activate now. I will let you know if I run across anything else.

**Comment by DavidCramer** [comment]
And I'll investigate it.
